notebooks/MAST/plotting_quasar_observations/plotting_quasar_observations.ipynb

```diff
@@ -0,0 +1,866 @@
+{
```

Comment:
<p>Can we provide a science motivation for why Hubble... for example, presumably the spectrum has been used in published work, what conclusions did they draw from it, and why? So you could say "If we want to investigate X phenomenon we'd want to study it in Y wavelength which we can do with Hubble" or something.</p><p> </p><p>Same comment as above for saying we need arrays instead of variables that are arrays.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>